### PR TITLE
Feature1 changes on scripts

### PR DESCRIPTION
**Pull request by carvajalemi**
Changes from Shei and me.

### CODE REVIEW

File: script.json.txt [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+name: Emilia Carvajal
+NickName: Emi
+age:  25
+address: Eudoro Galindo 1915
+department: 35C
```
Comment (Sheilasc) [inline script.json.txt:6]: this line is not good for me, please change

Comment (pablosangueza) [reply script.json.txt:6]: to me it is ok

**Review by Sheilasc — APPROVED** [review]

File: Script1.json.txt [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+name: Sheila Salinas
+age: 25
+address: Andres Manuel del rio
+depa: 3A
+changed: 1
+changed: 2
+changed: 3
```
Comment (pablosangueza) [inline Script1.json.txt:8]: review this, please change

File: script.json.txt [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+jobby 1: cook
+jobby 2: bake
+jobby 3: sleep
+jobby 4: play with Frida
+jobby 5: go to the gym
+jobby 6: travel
+city: Cochabamba
+Country: Bolivia
```
Comment (pablosangueza) [inline script.json.txt:14]: change this

**Review by pablosangueza — CHANGES_REQUESTED** [review]

**Review by pablosangueza — APPROVED** [review]

**Pull request merged**